app_backend/save_seat.py

```diff
@@ -0,0 +1,90 @@
+from sections import Sections
+
+from fastapi import APIRouter, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+from pydantic import BaseModel
+from sections import Sections
+#import server
+
+load_dotenv()
+db = Database(os.getenv('DB_USERNAME'), os.getenv('DB_PASSWORD'))
+db.connect()
+
+
+router = APIRouter(prefix="/seating")
+resp = Response
+
+seating_db = db.client["user"] # from user collection
+seating_collection = seating_db["seating"] # review --> sub collection
+
+
+
+class SaveSeating(BaseModel):
+    """
+        section: The seating section (numbered from 1 to 9)
+
+        color: The respective color associated with each seating section
+
+    """
+    section: int # required
+    username: str # required
+    color: str # not required
+
+
+@router.post("/dct-seating-section", status_code=status.HTTP_201_CREATED)
+async def set_section_info(resp: Response, seating_info: SaveSeating, dry_run) -> bool:
```

Comment:
`dry_run: bool = False` -- set the default to false so we don't get errors when its not included